solutions/client-setup/namespaces/client-name-networking.yaml

```diff
@@ -76,6 +76,23 @@ spec:
   role: roles/dns.admin
   member: "serviceAccount:client-name-networking-sa@client-management-project-id.iam.gserviceaccount.com" # kpt-set: serviceAccount:${client-name}-networking-sa@${client-management-project-id}.iam.gserviceaccount.com
 ---
+# Grant GCP role DNS Records Admin to GCP SA
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicyMember
+metadata:
+  name: client-name-networking-sa-dns-records-admin-permissions # kpt-set: ${client-name}-networking-sa-dns-records-admin-permissions
```

Comment:
client-name-networking-sa-tier2-dns-record-admin-permission

Reply:
Edited the wrong resource, but I think it's ok now.